.vscode/settings.json

```diff
@@ -16,6 +16,6 @@
         "typescript"
     ],
     "editor.codeActionsOnSave": {
-        "source.fixAll": true,
```

Comment:
This would affect the current using experience, suggest to add an option about this elsewhere and keep the current behavior